docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,111 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+Use this guide to create Pop!_OS bootable USB media from Linux:
+
+- Install ISO flashing software.
+- Download the Pop!_OS.iso file.
+- Verify the ISO.
+- Flash the ISO to your USB storage device.
+
+---
+## I. Install Popsicle
+
+![Popsicle Icon](/images/create-bootable-usb-linux/popsicle-icon.png)
+
+Create a bootable USB drive for Pop!_OS using Popsicle. Popsicle is a Linux utility for flashing multiple USB devices in parallel, written in Rust. You can install Popsicle via flatpak or AppImage.
```

Comment:
The first sentence here seems misplaced; it's a directive to create a bootable USB when the section is supposed to be about installing Popsicle.

Regarding the second sentence, I don't know that someone reading material at this level would know what "written in Rust" means or care.

For the third sentence, it looks like Flatpak's documentation capitalizes Flatpak, and I would recommend doing that as well: https://docs.flatpak.org/en/latest/introduction.html

Reply:
Agree on standardizing capitalization. 

On the Rust note, I would suggest something like:

"Popsicle is a Linux utility for flashing USB devices. The devices can be flashed one at a time, or in parallel. Popsicle can be installed via either Flatpak or AppImage.

>**NOTE:** Popsicle is written in the Rust programming language and is open-source. If interested, the source code can be viewed [here](link).

Reply:
@storyaddict idk about that. Making the callout even less subtle doesn't help if the question is whether or not the callout is relevant in the first place. Adding "note" to the beginning of it doesn't stop it from being a break in the flow of information.